src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.scss

```diff
@@ -0,0 +1,67 @@
+.form__wrapper {
+  font-family: var(--ubs-quaternary-font);
+  margin-bottom: 22px;
+}
+
+.form__headline {
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 28px;
+  letter-spacing: 0.002em;
+  color: var(--ubs-primary-grey);
+  margin-bottom: 30px;
+  margin-top: 36px;
+}
+
+.form__subtitle {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 24px;
+  color: var(--ubs-quaternary-dark-grey);
+  letter-spacing: 0.1px;
+}
+
+.form__label {
+  color: var(--ubs-secondary-grey);
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.form-group {
+  margin-bottom: 10px;
+}
+
+.btn {
+  color: var(--ubs-primary-green);
+  background: var(--ubs-primary-white);
+  width: 177px;
+  margin-bottom: 14px;
+  border: 1px solid var(--ubs-primary-green);
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+
+.violation__item {
+  margin-right: 38px;
+}
+
+input {
+  position: static;
+  height: 36px;
+  left: calc(50% - 340px / 2);
+  top: calc(50% - 36px / 2 + 10px);
```

Comment:
The same thing
I believe you can simplify those calculations

Reply:
Fixed